frontend/src/reviews/ReviewComments.tsx

```diff
@@ -31,23 +40,76 @@ export default function ReviewComments({ release, accessRequest }: ReviewComment
     ...semverOrAccessRequestIdObject,
   })
 
-  const reviewDetails = useMemo(
-    () =>
-      reviews.map((review) =>
-        review.responses.map((response) => <>{response.decision && <ReviewDecision response={response} />}</>),
-      ),
-    [reviews],
-  )
+  const reviewDetails = useMemo(() => {
+    let decisionsAndComments: Array<ReviewResponseKind> = []
+    reviews.forEach((review) => {
+      review.responses.forEach((response) => decisionsAndComments.push(response))
+    })
+    if (release && release.comments) {
+      decisionsAndComments = [...decisionsAndComments, ...release.comments]
+    }
+    decisionsAndComments.sort(sortByCreatedAtAscending)
+    return decisionsAndComments.map((response) => {
+      if (isReviewResponse(response)) {
+        return <ReviewDecisionDisplay key={response.createdAt} response={response as ReviewResponse} />
+      } else {
+        return <ReviewCommentDisplay key={response.createdAt} response={response as ReviewComment} />
+      }
+    })
+  }, [reviews, release])
+
+  async function submitReviewComment() {
+    if (release && currentUser) {
+      const updatedRelease: ReleaseInterface = release
+      if (!updatedRelease.comments) {
+        updatedRelease.comments = []
+      }
+      const newComment: ReviewComment = {
+        comment: newReviewComment,
+        user: currentUser.dn,
```

Comment:
This is insecure and would let users update other peoples comments or impersonate other people.  Instead, a 'createReviewComment' endpoint should be made which gets the user from the `req.user` field.  See the 'review' endpoint for how this is done.

In addition, the 'editRelease' functionality is also only available to model owners, so won't be available to other people.

Reply:
Oh duh I completely forgot about the edit model permission! Will update now.